src/XrdSciTokensHelper.hh

```diff
@@ -0,0 +1,56 @@
+
+#include <string>
+#include <vector>
+
+class XrdSciTokensHelper
+{
+public:
+
+//-----------------------------------------------------------------------------
+//! Initialize SciTokens plugin via authentication plugin path.
+//!
+//! @param   lp    - Pointer to the error logging object.
+//! @param   cfn   - Pointer to the configuration file used by xrootd.
+//! @param   parm  - Pointer to the plugin library parameters.
+//!
```

Comment:
You should document the accP and indicate that the "real" ztn will call this with a nullptr (it doesn't have an authorization object to give you).